airbyte-cdk/python/airbyte_cdk/connector_builder/message_grouper.py

```diff
@@ -36,8 +37,6 @@
 )
 from airbyte_protocol.models.airbyte_protocol import Type as MessageType
 
-JsonType = Optional[Union[str, int, float, bool, Dict[str, "JsonType"], List["JsonType"]]]
```

Comment:
moved to [types.py](airbyte-cdk/python/airbyte_cdk/sources/utils/types.py) so it can be shared